internal/pkg/ccloudv2/flink_gateway.go

```diff
@@ -106,10 +106,10 @@ func (c *FlinkGatewayClient) GetStatementResults(environmentId, statementId, org
 		req = req.PageToken(pageToken)
 	}
 	resp, httpResp, err := req.Execute()
-	return resp, errors.CatchCCloudV2Error(err, httpResp)
+	return resp, flink.CatchFlinkError(err, httpResp)
 }
 
 func (c *FlinkGatewayClient) GetExceptions(environmentId, statementId, orgId string) (flinkgatewayv1alpha1.SqlV1alpha1StatementExceptionList, error) {
 	resp, httpResp, err := c.StatementExceptionsSqlV1alpha1Api.GetSqlV1alpha1StatementExceptions(c.flinkGatewayApiContext(), environmentId, statementId).OrgId(orgId).Execute()
-	return resp, errors.CatchCCloudV2Error(err, httpResp)
+	return resp, flink.CatchFlinkError(err, httpResp)
```

Comment:
The current name is a bit redundant
```suggestion
	return resp, flink.CatchError(err, httpResp)
```

Reply:
done